google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
```

Comment:
This file should not be here any longer.

Reply:
Yes, somehow this got added back in.